src/views/List.jsx

```diff
@@ -1,10 +1,15 @@
 import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const navigate = useNavigate();
+
 	const [searchInput, setSearchInput] = useState('');
 	const [filteredData, setFilteredData] = useState([]);
 
+	const trueData = data.filter((item) => item.name !== null);
```

Comment:
This works great to filter out the placeholder!
What are your thoughts on filtering out the placeholder when making the API request?

Reply:
That's interesting and might make things a little cleaner, I'd love to test that out.

Reply:
We found a way to make it work!